src/VirtoCommerce.WebHooksModule.Core/Models/EventObjectProperties.cs

```diff
@@ -0,0 +1,8 @@
+namespace VirtoCommerce.WebhooksModule.Core.Models
+{
+    public class EventObjectProperties
+    {
+        public bool Discovered { get; set; }
+        public string[] Properties { get; set; }
```

Comment:
Use List instead of array, please.
```suggestion
        public List<string> Properties { get; set; }
```